<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendReplyDialog</class>
 <widget class="QDialog" name="SendReplyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textView">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SendReplyTextEdit" name="replyEdit">
     <property name="tabChangesFocus">
      <set>true</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SendReplyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SendReplyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
  <customwidgets>
  <customwidget>
   <class>SendReplyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>sendreplydialog.h</header>
  </customwidget>
 </customwidgets>
</ui>
